api/securitytest/run.go

```diff
@@ -161,10 +162,13 @@ func (results *RunAllInfo) runLanguageScans(enryScan SecTestScanInfo) error {
 					return
 				}
 			}
+			// if err := newLanguageScan.Start(); err == nil {
 			if err := newLanguageScan.Start(); err != nil {
+				results.Containers = append(results.Containers, newLanguageScan.Container)
 				select {
 				case <-syncChan:
 					return
+					// case errChan <- errors.New("ERRROR"):
```

Comment:
Would you mind removing this comment as well? 🙃

Reply:
Removed.